packages/editor-ui/src/components/MainSidebar.vue

```diff
@@ -425,7 +425,8 @@ export default mixins(
 
 					saveAs(blob, workflowName + '.json');
 				} else if (key === 'workflow-save') {
-					this.saveCurrentWorkflow(undefined);
+					await this.saveCurrentWorkflow(undefined);
+					this.$store.dispatch('settings/fetchPromptsData');
```

Comment:
this goes back to the original issue now... what should happen if the save fails? if we want to not show the prompt, then we need to handle it
```suggestion
					const saved = await this.saveCurrentWorkflow(undefined);
					if (saved) { 										 
                                              {this.$store.dispatch('settings/fetchPromptsData');
					}
```

Reply:
Fixed